src/decoders/arw.rs

```diff
@@ -73,7 +73,7 @@ impl<'a> Decoder for ArwDecoder<'a> {
 }
 
 impl<'a> ArwDecoder<'a> {
-  fn image_a100(&self, camera: &Camera) -> Result<RawImage,String> {
```

Comment:
Do you mean `ok_image`, not `ok_camera`? Previously the hash map in `RawLoader` ensured the instance was kept alive. I'm a bit inexperienced with lifetimes, but it seems to me to be problematic handle it, and making `ok_image` conuming it when it extracts its values seemed reasonable.